### PR TITLE
Completed html-forms exercise

### PR DESCRIPTION
**Pull request by wang-morris**
Screen Shot 2022-05-12 at 8 10 14 PM

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    <label for="question">Why is coding cool?</label><br />
+    <select name="coding-question" id="question">
+      <option value="select-option">Select an option</option>
+      <option value="interesting">Because it's interesting</option>
+      <option value="fun">Because it's fun</option>
+      <option value="powerful">Because it's powerful</option>
+    </select>
+  </div><br />
+  <div class="coding-form">
+    <label for="favorite-language">Favorite Language (choose one)</label><br /><br />
```
Comment (BrettAlbright) [inline html-forms/index.html]: make sure to avoid using `br` tags. you can use a `p` tag for this text

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    <input type="radio" name="favorite-language" value="html" checked="checked" />HTML
```
Comment (BrettAlbright) [inline html-forms/index.html]: each of these inputs should have a label, you can wrap the labels around the input if desired

File: html-forms/index.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    <input type="radio" name="favorite-language" value="css" />CSS
+    <input type="radio" name="favorite-language" value="javascript" />JavaScript
+  </div>
+  <div class="coding-form"><br />
+    <label for="information-checkbox">Would you like to sign up for more information?</label><br /><br />
+    <input type="checkbox" name="information-checkbox" value="sign-up" />Sign me up!
```
Comment (BrettAlbright) [inline html-forms/index.html]: make sure the input has an id attribute which matches the `for` attribute of the label

File: html-forms/index.html [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+  </div>
+  <div class="coding-form">
+    <label for="comments">Comments</label><br />
+    <textarea name="comments" cols="30" rows="10"></textarea>
```
Comment (BrettAlbright) [inline html-forms/index.html]: id attribute needed here

File: html-forms/index.html [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+  </div>
+  <div class="coding-form">
+    <label for="submit-button"></label>
+    <input type="submit" name="submit-button" value="Submit"/>
```
Comment (BrettAlbright) [inline html-forms/index.html]: you don't need a label for the submit input

File: html-forms/index.html [hunk 6]
```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Html Forms</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+  <div class="coding-form">
+    <label for="email">Email</label><br />
```
Comment (BrettAlbright) [inline html-forms/index.html]: make sure to get rid of this `br` tag

**Review by BrettAlbright — CHANGES_REQUESTED** [review]
on its way, some updates are needed

File: html-forms/index.html [hunk 7]
```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Html Forms</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+  <div class="coding-form">
```
Comment (scrumptiousdev) [inline html-forms/index.html]: I think we are missing some important html element here
HINT: this exercise is `html-forms`

**Comment by wang-morris** [comment]
Screen Shot 2022-05-12 at 9 41 16 PM

**Review by BrettAlbright — APPROVED** [review]
👍🏼

**Pull request merged**